src/services/user-group.service.ts

```diff
@@ -0,0 +1,37 @@
+import { Sequelize } from 'sequelize';
+import { ModelCtor } from 'sequelize/types';
+import { Inject, Service } from 'typedi';
+
+import { Config } from '../config';
+import { IGroupInstance, IUserInstance } from '../models';
+import { EntitiesOperationStatus } from './models';
+
+@Service()
+export class UserGroupService {
+    constructor(
+        @Inject(Config.injectionToken.groupModel) private groupModel: ModelCtor<IGroupInstance>,
+        @Inject(Config.injectionToken.userModel) private userModel: ModelCtor<IUserInstance>,
+        @Inject(Config.injectionToken.sequelize) private sequelize: Sequelize,
+    ) {}
+
+    async addUsersToGroup(groupId: string, usersIds: string[]) {
+        try {
+            return await this.sequelize.transaction(async () => {
+                const group = await this.groupModel.findByPk(groupId);
```

Comment:
I don't see usage of the transactions here. Please refer to the [manual](https://sequelize.org/master/manual/transactions.html)

Reply:
@KonanMentor  
If I understood right, there is an ability to automatically pass transactions to all queries:
![image](https://user-images.githubusercontent.com/33813087/94335190-8fe40a80-ffe2-11ea-8726-e47bcd535f5a.png)
I did all instructions from above (for not passing transactions manually to all queries):

![image](https://user-images.githubusercontent.com/33813087/94335279-287a8a80-ffe3-11ea-9f6b-8e267894ee99.png)
So as I understand it is managed transaction (we don't need to manually commit and rollback) and transactions are automatically passed to all queries, am I wrong?

Reply:
Ok, nice